src/main/scala/viper/carbon/verifier/BoogieInterface.scala

```diff
@@ -138,10 +143,12 @@ trait BoogieInterface {
     var res: String = ""
     var reserr: String = ""
     def out(input: java.io.InputStream): Unit = {
+      reporter report BackendSubProcessReport("carbon", boogiePath, onOutput/*, _boogieProcessPid*/)
```

Comment:
It's a bit strange that "onOutput" is lower case, while the other stages are upper case (such as "OnError"). But this is related to https://github.com/viperproject/silver/pull/513.